apps/service_gather/test/gather/extraction_test.exs

```diff
@@ -15,7 +15,7 @@ defmodule Gather.ExtractionTest do
   Temp.Env.modify([
     %{
       app: :service_gather,
-      key: Gather.Extraction,
+      key: Gather.Extraction.SourceStream.SourceHandler,
```

Comment:
Feels like SourceStream has no meaning in this module name now.

Reply:
Agreed

Reply:
Moved the only `SourceStream` function back up to `Extraction` and then got rid of the empty `SourceStream` wrapper module so now it's just `Gather.Extraction.SourceHandler`.